app/server.js

```diff
@@ -1,30 +1,35 @@
-const Hapi = require('@hapi/hapi')
-const Joi = require('joi')
-const { registerPlugins } = require('./plugins/index.js')
-const config = require('./config.js')
+import hapi from '@hapi/hapi'
+import joi from 'joi'
+import blipp from 'blipp'
```

Comment:
This library annoys me by filling my console with a list of routes each time I restart the app, for what purpose I'm not sure. Shall we just dump it @aaroncarroll82, @bendef-24 and @omega1119?

Reply:
Yeah it seems like thats the sole purpose of the library (https://www.npmjs.com/package/blipp)

Reply:
'are' loads 🤦

Reply:
Even for the journey apps I don't see the point, I don't need to see a big list of all the app endpoints in my console every time I restart the app. If I need to know what routes we have I can look in the routes file.

Reply:
I've removed Blipp from this repo